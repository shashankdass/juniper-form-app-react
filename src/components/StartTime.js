import React, { Component } from 'react';
import { Col, FormGroup, Label, Input } from 'reactstrap';

class StartTime extends Component{
  render() {
    return(
      <div>
        <FormGroup row>
          <Label sm={2}>
            Start Time:
            </Label>
            <Col sm={5}>
              <Input name="startTime" type="text" />
            </Col>
        </FormGroup>
      </div>
    )
  }
}

export default StartTime;
