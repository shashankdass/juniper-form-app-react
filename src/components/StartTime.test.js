import React from 'react';
import StartTime from './StartTime';
import renderer from 'react-test-renderer';

test('it renders StartTime Component', () => {
  const component = renderer.create(
    <StartTime></StartTime>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
