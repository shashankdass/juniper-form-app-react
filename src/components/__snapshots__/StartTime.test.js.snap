exports[`test it renders StartTime Component 1`] = `
<div>
  <div
    className="row form-group">
    <label
      className="col-sm-2 col-form-label"
      htmlFor={undefined}>
      Start Time:
    </label>
    <div
      className="col col-sm-5">
      <input
        className="form-control"
        name="startTime"
        type="text" />
    </div>
  </div>
</div>
`;
